deployment/dockerfiles/origin-growth

```diff
@@ -4,13 +4,22 @@ WORKDIR /app
 
 ENV NODE_ENV=production
 
+# Install envkey-source to make environment available for sequelize migration
+RUN curl -s -L -o envkey-source.tar.gz https://github.com/envkey/envkey-source/releases/download/v1.2.5/envkey-source_1.2.5_linux_amd64.tar.gz
+RUN tar -zxf envkey-source.tar.gz 2> /dev/null
+RUN rm envkey-source.tar.gz
+RUN mv envkey-source /usr/local/bin
+
 COPY package*.json ./
 COPY lerna.json ./
-COPY ./origin-js ./origin-js
+COPY ./experimental/origin-graphql/ ./experimental/origin-graphql/
 COPY ./origin-growth ./origin-growth
 COPY ./origin-identity ./origin-identity
 COPY ./scripts ./scripts
 
 RUN npm install --unsafe-perm
 
-CMD npm run start:growth --prefix origin-growth
\ No newline at end of file
+# Run migrations then start the growth server.
+CMD eval $(envkey-source) && \
+	npm run migrate --prefix origin-growth && \
+	npm run start:growth --prefix origin-growth
```

Comment:
Not really related to this PR but I don't think we need the `npm run start:growth`, it should just be `npm run start` for everything. I think we should clean this up at some point.

Reply:
Fixed.